src/lib/data/createables.ts

```diff
@@ -2189,6 +2189,15 @@ const Createables: Createable[] = [
 			[itemID("Tarn's diary")]: 1
 		}
 	},
+	{
+		name: 'Ring of wealth (i)',
+		inputItems: new Bank({
+			'Ring of wealth': 1,
+			'Ring of wealth scroll': 1
+		}),
+		GPCost: 50_000,
+		outputItems: { [itemID('Ring of wealth (i)')]: 1 }
```

Comment:
All new usages of banks should be, always, `new Bank().add().add()` (these 2 methods used here are deprecated). You don't have to change it, but its just a note for the future.